technique/reflectometry/base.py

```diff
@@ -255,6 +255,59 @@ def slit_check(theta, footprint, resolution):
     print("s2vg={}".format(s2))
 
 
+def auto_height(laser_block: str, fine_height_block: str, target: float = 0.0, continue_if_NaN: bool=False):
```

Comment:
In run_angle there is a parameter called auto_height which shadows this name, it then subsequently calls the auto_height function but without params and with a TODO next to it to write it. I suggest we change one of the names e.g. call one of the do_auto_height.